modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -74,8 +74,17 @@ def validate_veteran_identifiers(require_birls: false) # rubocop:disable Metrics
         end
 
         mpi_add_response = target_veteran.mpi.add_person_proxy
+        ids = target_veteran.mpi.participant_ids
```

Comment:
assuming we're duplicating [this line](https://github.com/department-of-veterans-affairs/vets-api/blob/c594dc2e38bb99754d5f29db7ebdb05c159e4630/modules/claims_api/app/controllers/claims_api/v1/application_controller.rb#L29) because it's after the `add_person_proxy` which may create the participant id?

Reply:
Yea, we could remove this and the next line (and move the `if ids.size.zero?` up, but I feel like these are good here so they give `add_person_proxy` a chance to get an ID.